README.md

```diff
@@ -49,8 +49,8 @@ To register a logback appender, create a new logback-spring.xml (or logback.xml)
     </encoder>
   </appender>
   <appender name="OpenTelemetry"
-            class="io.opentelemetry.instrumentation.logback.appender.v1_0.OpenTelemetryAppender"
-            captureExperimentalAttributes="true">
+            class="io.opentelemetry.instrumentation.logback.appender.v1_0.OpenTelemetryAppender">
+    <captureExperimentalAttributes>true</captureExperimentalAttributes>
```

Comment:
This looks fine, but just wondering if it is worth noting in the readme the impact of setting `captureExperimentalAttributes` to true.  What exactly does experimental attributes include?  Is there a link we could include so users could read more?